apps/web/app/environments/[environmentId]/people/page.tsx

```diff
@@ -1,13 +1,14 @@
-import PeopleList from "./PeopleList";
+export const revalidate = 0;
+
+import { Suspense } from "react";
+import PeopleList from "@/app/environments/[environmentId]/people/PeopleList";
+import Loading from "@/app/environments/[environmentId]/people/Loading";
 
 export default function PeoplePage({ params }) {
   return (
-    <>
-      <h1 className="my-2 text-3xl font-bold text-slate-800">People</h1>
-      <p className="mb-6 text-slate-500">
-        A list of all people who used your application since embedding the Formbricks JS widget.
-      </p>
+    <Suspense fallback={<Loading />}>
```

Comment:
please remove the suspense and use the integrated next.js logic for that
https://nextjs.org/docs/app/building-your-application/routing/loading-ui-and-streaming#instant-loading-states

(sorry for the back and forth, I'm also still learning :-D )